examples/whisper/requirements.txt

```diff
@@ -1,5 +1,4 @@
-coloredlogs
-flatbuffers
 onnx==1.13.1
+onnxruntime==1.15.0
```

Comment:
need we pin the version or just use the latest ort version?

Reply:
We need to pin it right now. It needs at least 1.15.0 but there are some new changes in ort main branch that are not compatible with what we have right now. 

I am planning to make the required changes to support versions > 1.15.0 (latest nightly) in a follow up PR.